src/ml_tooling/transformers/rare_feature.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class RareFeatureEncoder(BaseEstimator, TransformerMixin):
+    """
+    Replace values which occurs below the chosen threshold.
+    The threshold can be a procent or int value.
+    The values are replaced with the chosen fill value.
+    Column names need to be the same.
+    """
+
+    def __init__(self, threshold=0.2, fill_rare="Rare"):
```

Comment:
Remember your type annotations:

```suggestion
    def __init__(self, threshold: Union[int, float] = 0.2, fill_rare: Any = "Rare"):
```

Reply:
Missing docstring for the `__init__`